web/pgadmin/misc/bgprocess/processes.py

```diff
@@ -608,6 +630,10 @@ def status(self, out=0, err=0):
                 'process_state': self.process_state
             }
 
+        # Set error message based on exit code if stderr is empty.
+        if err_completed and len(stderr) == 0:
+            set_error_msg(self.cmd, self.ecode, stderr)
```

Comment:
I would suggest to make it as get_error_msg and then append  the returned message to stderr here.

Reply:
Done